ScoutSuite/providers/azure/rules/findings/storageaccount-soft-delete-enabled.json

```diff
@@ -0,0 +1,27 @@
+{
+    "description": "Storage soft delete is enabled",
```

Comment:
All words start with an uppercase - you can just run https://github.com/nccgroup/ScoutSuite/tree/master/tools#format_findingspy to take care of formatting.

Reply:
Done